include/plasma_types.h

```diff
@@ -7,8 +7,8 @@
  *  University of Manchester, UK.
  *
  **/
-#ifndef PLASMA_TYPES_H
-#define PLASMA_TYPES_H
+#ifndef ICL_PLASMA_TYPES_H
```

Comment:
All global macros must be prefixed with PLASMA not ICL

Reply:
Corrected.